.yarn/versions/52395a7b.yml

```diff
@@ -0,0 +1,22 @@
+releases:
+  "@yarnpkg/cli": patch
+  "@yarnpkg/plugin-compat": patch
```

Comment:
`@yarnpkg/plugin-compat` doesn't need a release for this change

Reply:
Fixed.